internal/plop-templates/react-native/README.md.hbs

```diff
@@ -0,0 +1,20 @@
+# {{titleCase name}} (React Native)
+
+TODO: Add description
+
+## Installation
+
+```bash
+$ npm install @vygruppen/spor-{{kebabCase name}}-react-native
+```
+
+## Usage
+
+```tsx
+import { } from "@vygruppen/spor-{{kebabCase name}}-react-native";
+```
+TODO: Add usage description
+
+## Development
+
+Please refer to the root readme for development notes.
```

Comment:
This is the README template that's added to all React Native projects. In the following two files, you'll find matching files for React and TypeScript projects as well.